modules/windows-containers-release-notes-limitations.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/windows-containers-release-notes-#-x
+
+[id="windows-containers-release-notes-limitations_{context}"]
+= Known limitations
+
+Note the following limitations when working with Windows nodes managed by the WMCO (Windows nodes):
+
+* The following {product-title} features are not supported on Windows nodes:
+** Red Hat OpenShift Developer CLI (odo)
+** Image builds
+** OpenShift Pipelines
+** OpenShift Service Mesh
+** OpenShift monitoring of user-defined projects
+
+* The following Red Hat features are not supported on Windows nodes:
+** link:https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/getting_started_with_cost_management/assembly-introduction-cost-management?extIdCarryOver=true&sc_cid=701f2000001OH74AAG#about-cost-management_getting-started[Red Hat cost management]
+** link:https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local]
+
+* Windows nodes do not support pulling container images from private registries. It is recommended that you use images from public registries or pre-pull the images.
```

Comment:
```suggestion
* Windows nodes do not support pulling container images from private registries. You can use images from public registries or pre-pull the images.
```
Maybe can be worded like this to avoid the "recommended". This wording is similar what you have in the following bullet. Up to you though!

Reply:
@kelbrown20 Good idea. Believe it or not, _it is recommended_ is the recommended phrase for us to use.

Reply:
Yeah that was always confusing for me when I'm going through the guides 😄